less/index.less

```
@@ -1,12 +1,12 @@
 // Import your less files here:
 
-@import: "variables.less";
-@import: "mixins.less";
-@import: "reset.less";
-@import: "global.less";
-@import: "navigation.less";
-@import: "footer.less";
-@import: "home-page.less";
+@import "variables.less";
```

```suggestion 
@import "variables"
```
you can drop the less as it assumes it is less by default